src/lib.rs

```diff
@@ -37,7 +37,7 @@
 #![cfg_attr(all(test, feature = "unstable"), feature(test))]
 #[cfg(all(test, feature = "unstable"))] extern crate test;
 
-#[cfg(any(test, feature="std"))] pub extern crate core;
+#[cfg(any(test, feature="std"))] extern crate core;
```

Comment:
This is technically a breaking change, although it seems to have been a bug in the first place to re-export core.